From a392d416cce022e2bec1ae7f2cadbeb5b579d158 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Fri, 19 Jun 2020 21:36:15 +0800
Subject: [PATCH] MALI: rockchip: Fix make distclean

make distclean
scripts/Makefile.clean:15: drivers/gpu/arm/midgard_for_linux/Makefile:
No such file or directory

Fixes: 25bb95267761 ("MALI: rockchip: remove drivers/gpu/arm/midgard_for_linux/")
Change-Id: Ia47e2581d9679f3bcca809ad4506b61068e1710d
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/gpu/arm/Kbuild | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/arm/Kbuild b/drivers/gpu/arm/Kbuild
index 5fdf2044b785..b3466f167d8e 100644
--- a/drivers/gpu/arm/Kbuild
+++ b/drivers/gpu/arm/Kbuild
@@ -20,7 +20,7 @@
 #
 
 
-obj-$(CONFIG_MALI_MIDGARD_FOR_LINUX) += midgard_for_linux/
+# obj-$(CONFIG_MALI_MIDGARD_FOR_LINUX) += midgard_for_linux/
 
 obj-$(CONFIG_MALI_MIDGARD_FOR_ANDROID) += midgard/
 
-- 
2.35.3

